### PR TITLE
[minor] add user availability to UserStatus

### PR DESCRIPTION
**Pull request by costela**
This minor PR adds support for the `availability` field in `UserStatus`, added in 13.6.0.

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -925,9 +925,10 @@ func (s *UsersService) GetUserActivities(opt *GetUserActivitiesOptions, options
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/users.html#user-status
 type UserStatus struct {
-	Emoji       string `json:"emoji"`
-	Message     string `json:"message"`
-	MessageHTML string `json:"message_html"`
+	Emoji        string `json:"emoji"`
+	Availability string `json:"availability"`
```
Comment (svanharmelen) [inline users.go:929]: Is there a fixed list of availability values? If so we should probably make a `const` for it instead of using a `string`

Comment (costela) [reply users.go:929]: good point. I tried to find one (for about 5 minutes :see_no_evil:) and came up empty.
I'll see what gitlab's code has to say about it :+1:

Comment (costela) [reply users.go:929]: ok, it seems gitlab [currently has only two possible options](https://gitlab.com/gitlab-org/gitlab-foss/-/blob/3cfe2ab491cdaf46d5a8ff6359fc6a89a7557eea/app/models/user_status.rb#L22): `not_set` and `busy`.

added as consts :heavy_check_mark:

**Comment by costela** [comment]
FYI: the API docs for _setting_ the availability is [currently do not mention the `availability` parameter](https://docs.gitlab.com/ce/api/users.html#set-user-status), but it works anyway, so I included it in the PR.

**Review by svanharmelen — APPROVED** [review]
Thanks @costela! Made a few last tweaks, but looks good now 👍🏻

**Pull request merged**